resources/dicts/relationship_events/DE_IN_CREASE/DECREASE_HIGH.json

```diff
@@ -20,39 +21,28 @@
         ],
         "like": [
             "is telling jokes about (cat).",
-            "whines about (cat).",
-            "is tired from (cat) putting them on so many patrols.",
-            "is bossing (cat) around.",
             "started a fight with (cat).",
             "has a fight with (cat) about what's right.",
+            "is not backing down in an argument with (cat).",
+            "scorns (apprentice) for not catching enough prey.",
```

Comment:
I'm not sure if someone changed the code, if not (apprentice) has to be (cat)

Reply:
I'll fix all instances of this right now